staging/src/k8s.io/apiserver/pkg/endpoints/filters/metrics.go

```diff
@@ -56,7 +56,7 @@ var (
 			Help:           "Counter of authenticated attempts.",
 			StabilityLevel: metrics.ALPHA,
 		},
-		[]string{"result"},
+		[]string{"result", "authenticator"},
```

Comment:
This metric is call `authentication_attempts` but it is a counter and counters should be suffixed with `_total`. Would you mind renaming this?